software/local-content-share.yml

```
@@ -0,0 +1,12 @@
+name: Local Content Share
+website_url: https://github.com/Tanq16/local-content-share
+description: Browser frontend that enables sharing and storing of text snippets and files with any local network devices. 
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Go
+tags:
+  - File Transfer - Peer-to-peer Filesharing
+  - File Transfer - Web-based File Managers
```

```suggestion
  - Pastebins
  - File Transfer - Single-click & Drag-n-drop Upload
```

Isn't that more accurate?